tests/ui/test_ui_tools.py

```diff
@@ -439,13 +439,31 @@ def test_mouse_event(self, mocker, stream_view):
         col = 1
         row = 1
         focus = "WIDGET"
-        # Left click
+        # Scroll wheel up: 4.
         stream_view.mouse_event(size, "mouse press", 4, col, row, focus)
-        stream_view.keypress.assert_called_once_with(size, "up")
+        stream_view.keypress.assert_called_once_with(
+            size, next(iter(keys_for_command('GO_UP')))
+        )
 
-        # Right click
+        # Scroll wheel down: 5.
         stream_view.mouse_event(size, "mouse press", 5, col, row, focus)
-        stream_view.keypress.assert_called_with(size, "down")
+        stream_view.keypress.assert_called_with(
+            size, next(iter(keys_for_command('GO_DOWN')))
+        )
```

Comment:
There's some independent copy/pasted code in this test - can you see how to refactor it? This would make an actual followup test-commit easier. Do you see what the comments could translate to?

Reply:
I have refactored this and a similar test in `UsersView` in a3ee7387670a70d9a5c7082f031c742c5134e0ed. 

I am afraid, I am not sure about what the comments could translate to mean?

Reply:
Well, what does 'mouse press, 4, up' mean? In a parametrized test, you can use custom ids, which I think you've already seen.

It may be worth checking that the comments are correct, as I'm not sure if 4 means 'left click'.

Reply:
I have added the appropriate test ids, the comments weren't correct. I referred to urwid's docs to find out the meanings.